src/network.c

```diff
@@ -1887,7 +1886,6 @@ static sockent_t *sockent_create(int type) /* {{{ */
   se->service = NULL;
   se->interface = 0;
   se->next = NULL;
-  pthread_mutex_init(&se->lock, NULL);
```

Comment:
was this removed completely by accident?

Reply:
As you can see, that field was fully removed from structure.